files/count.pl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/perl
```

Comment:
Do you need a local copy of count.pl? It looks like you download it from upstream elsewhere.